thread.c

```diff
@@ -993,13 +1001,20 @@ rb_thread_create(VALUE (*fn)(void *), void *arg)
 VALUE
 rb_thread_create_ractor(rb_ractor_t *r, VALUE args, VALUE proc)
 {
+    VALUE thgroup = r->thgroup_default = rb_obj_alloc(cThGroup);
```

Comment:
init at `thread_do_start_proc()` seems better.

Reply:
Ok, I'll change it.

Reply:
https://github.com/ruby/ruby/pull/13016